weblate/trans/models/project.py

```diff
@@ -569,6 +569,11 @@ def all_admins(self):
 
         return User.objects.all_admins(self).select_related("profile")
 
+    def all_reviewers(self):
+        from weblate.auth.models import User
+
```

Comment:
Avoid hitting the database if reviews are not enabled:

```suggestion
        if not self.enable_review:
            return User.objects.none()
```